modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/constants/ClaySamplePortletKeys.java

```diff
@@ -27,6 +27,9 @@ public class ClaySamplePortletKeys {
 	public static final String NAVIGATION_BARS_DISPLAY_CONTEXT =
 		"NAVIGATION_BARS_DISPLAY_CONTEXT";
 
+	public static final String SEARCH_PAGINATOR_DISPLAY_CONTEXT =
+		"SEARCH_PAGINATOR_DISPLAY_CONTEXT";
+
```

Comment:
These bits should be removed from here as search paginator will be sampled in the other module

Reply:
Done!